x-pack/test/functional/apps/security/doc_level_security_roles.js

```diff
@@ -74,7 +74,7 @@ export default function ({ getService, getPageObjects }) {
         expect(hitCount).to.be('1');
       });
       const rowData = await PageObjects.discover.getDocTableIndex(1);
-      expect(rowData).to.be('name:ABC Company region:EAST _id:doc1 _type:_doc _index:dlstest _score:1');
+      expect(rowData).to.be('name:ABC Company region:EAST _id:doc1 _type:_doc _index:dlstest _score:0');
```

Comment:
Why'd the score change to 0?

Reply:
At the root of the ES query we generate there's a `bool` query. Historically the Lucene query has gone under the `must` clause, which does scoring. That added unnecessary work for ES since we don't sort by score. KQL queries go under the `filter` clause to avoid that unnecessary scoring.

Reply:
@Bargs I'm coming across this now...how should one do scoring when searching with KQL? Is it no longer possible to have resulted sorted according to the ES score when searching with KQL?

Reply:
Correct, KQL queries do not participate in scoring.